extensions/query-engine/src/main/java/com/aerospike/helper/query/Qualifier.java

```diff
@@ -302,17 +302,20 @@ public String luaFilterString(){
 		FilterOperation op = getOperation();
 		switch (op) {
 			case AND:
-				return new StringBuffer()
+				return new StringBuilder()
```

Comment:
StringBuffer is thread safe implementation of StringBuilder. Synchronization is obviously not needed here